### PR TITLE
Release/1.2.19

### PR DESCRIPTION
**Pull request by lawrence-forooghian**

### CODE REVIEW

File: Spec/Tests/RealtimeClientConnectionTests.swift [hunk 1]
```diff
@@ -2772,7 +2772,7 @@ class RealtimeClientConnectionTests: XCTestCase {
         }
     }
 
-    // FIXME: Fix flaky presence tests and re-enable. See https://ably-real-time.slack.com/archives/C030C5YLY/p1.2.182436085700
+    // FIXME: Fix flaky presence tests and re-enable. See https://ably-real-time.slack.com/archives/C030C5YLY/p1.2.192436085700
```
Comment (lawrence-forooghian) [inline Spec/Tests/RealtimeClientConnectionTests.swift:2775]: Hmm, this is an issue with the bump_patch script. Will fix.

Comment (lawrence-forooghian) [reply Spec/Tests/RealtimeClientConnectionTests.swift]: d3945fc37b22aaafa29fac66ba94a9df6e44851e

**Review by QuintinWillison — APPROVED** [review]

File: Spec/Tests/RealtimeClientConnectionTests.swift [hunk 2]
```diff
@@ -2772,7 +2772,7 @@ class RealtimeClientConnectionTests: XCTestCase {
         }
     }
 
-    // FIXME: Fix flaky presence tests and re-enable. See https://ably-real-time.slack.com/archives/C030C5YLY/p1.2.182436085700
+    // FIXME: Fix flaky presence tests and re-enable. See https://ably-real-time.slack.com/archives/C030C5YLY/p1623172436085700
```
Comment (maratal) [inline Spec/Tests/RealtimeClientConnectionTests.swift:2775]: I think this is an accident

Comment (lawrence-forooghian) [reply Spec/Tests/RealtimeClientConnectionTests.swift:2775]: Did you see this commit @maratal? https://github.com/ably/ably-cocoa/pull/1558/commits/d3945fc37b22aaafa29fac66ba94a9df6e44851e

**Review by maratal — APPROVED** [review]

**Pull request merged**